plot_aggregate_loanbooks.R

```diff
@@ -38,6 +38,12 @@ if (file.exists(here::here(".env"))) {
     output_path_aggregated <- file.path(output_path, sector_split_type_select, "aggregated")
   }
 
+  by_groups <- Sys.getenv("BY_GROUP")
```

Comment:
Ah, you probably just need to rename this `by_groups -> by_group` lol (relating to my later comment)

Reply:
yea, this whole naming thing is a bit odd.. in `run_aggregate_loanbooks.R` you can have more than 1 `by_group`. In `plot_aggregate_loanbooks.R` you currently can't. But maybe for now it's fine to have slightly different names although they get the input from the same config entry.

Reply:
Hmmm that seems dangerous (having the same config input parameter have different requirements/ conditions depending on where it's used)

Reply:
It's not technically going to fail if you pass it more than one value in `plot_aggregate_loanbooks.R` it will just print messages that the plots cannot be created for more than one `by_group`

Reply:
which is more due to the fact that we have not really had time to decide where the limits of the existing plots are/should be

Reply:
I opted for streamlining the name to `by_group` across both scripts and add an explanation to the `README`

Reply:
Sounds good@